samples/bluetooth/direction_finding_connectionless_tx/overlay-aod.conf

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
```suggestion
# Copyright (c) 2024 Nordic Semiconductor ASA
```

Reply:
The file was renamed and content modified, is this necessary?

Reply:
Oh, I thought it is new. Ignore this.